sql/scalar-functions.mdx

```diff
@@ -0,0 +1,333 @@
+---
+title: Scalar Functions
+description: 'Supported Scalar Functions in Arroyo SQL'
+---
+
+Arroyo's Scalar function support mainly mirrors that of DataFusion, 
+which in turn uses the semantics of the equivalent Postgres functions.
+
+### Math Functions
```

Comment:
I think every header should be reduced by a level (so these would have `##`, functions would have `###`) so that it renders better in the ToC:

![image](https://github.com/ArroyoSystems/arroyo-docs/assets/100920/4629a8ab-0e86-47a2-8309-8eb002470e97)